modin/experimental/engines/omnisci_on_ray/frame/data.py

```diff
@@ -1203,7 +1206,16 @@ def from_pandas(cls, df):
             orig_df.index.names = orig_index_names
         new_dtypes = df.dtypes
         df = df.add_prefix("F_")
-        new_parts, new_lengths, new_widths = cls._frame_mgr_cls.from_pandas(df, True)
+
+        partitions, unsupported_cols = cls._frame_mgr_cls.from_pandas(df, True)
```

Comment:
Can we add one more value to the returned tuple instead of packing tuple into another tuple?

Reply:
done